www/src/components/TopBar/components/Nav.tsx

```diff
@@ -11,6 +11,9 @@ const Nav: React.FC = () => {
       <StyledLink exact activeClassName="active" to="/farms">
         Menu
       </StyledLink>
+      <StyledLink exact activeClassName="active" to="/nfts">
+        NFTs
```

Comment:
I have doubts that we need it at the moment without at least some "coming soon" dummy page

Reply:
Yea. Maybe a good idea would be to just have a simple page that displays "COMING SOON"

Reply:
I believe Sergei works on dummy nft page ATM - let's wait for him